examples/test_scr_poststage

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+#
+# This script generates checkpoints on the IBM burst buffer disk, and then
+# starts transferring them to GPFS before exiting.  Then, the scr_poststage
+# script gets run after the transfers finish, and finalizes them.
+#
+# Example:
+#
+# # Allocate one node with 100GB burst buffer, run test_scr_poststag to
+# # create and begin transferring checkpoints, and after the transfers finish
+# # (after the job ends), call scr_poststage to finalize them.
+# bsub -nnodes 1 -stage storage=100:out=,scr_poststage lexec jsrun -r 1 test_scr_poststage
```

Comment:
Let's drop lexec if that still works, since lexec is specific to LLNL.
```suggestion
# bsub -nnodes 1 -stage storage=100:out=,scr_poststage jsrun -r 1 test_scr_poststage
```